daml.yaml

```diff
@@ -5,6 +5,8 @@ init-script: Demo:demo
 parties:
   - Alice
   - Bob
+  - Dana
+  - Carl
```

Comment:
Do we use these parties in our tests? They are newly introduced in this PR, and the only other place I see them referenced is in `getExistingParties`. Can they be dropped?

Reply:
They are used elsewhere in `Tests.Clearing.Setup` and I think `Distribution.Auction` (although I see now it's `Charlie` not `Carl`, so I will change). They aren't used in every test, but they are allocated in all tests via `allocateParties`. Not sure if it really helps or hurts having them all here, I believe the only thing it does is populate the navigator dropdown.

Reply:
Yep makes sense, it doesn't hurt to have them if they are used in tests